front/src/index.html

```diff
@@ -1,5 +1,6 @@
 <html>
     <head>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
what is that ?

Reply:
for icons but i would recomend [this](https://feathericons.com/) because open source

Reply:
is a link to import googles icon than I have used

Reply:
use the open source one's proposed by js-all

Reply:
migth also be better to include the font in the project sources directly

Reply:
okay

Reply:
don't need that anymore since you put the images in icons file